### PR TITLE
Use github actions

### PR DESCRIPTION
**Pull request by hikhvar**

### CODE REVIEW

**Review by corny — DISMISSED** [review]

**Review by corny — APPROVED** [review]

**Review by corny — DISMISSED** [review]

**Comment by corny** [comment]
@taktv6 please merge

File: .github/linters/.golangci.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+run:
+  skip-dirs:
+    - net/tcp
```
Comment (hikhvar) [inline .github/linters/.golangci.yml:3]: Why do we exclude this?

**Review by corny — DISMISSED** [review]

File: .github/linters/.golangci.yml (hunk 1 above)
Comment (corny) [reply .github/linters/.golangci.yml:3]: I was not able to fix the errors. If you have some time, give it a try:

https://github.com/bio-routing/bio-rd/runs/1187348144?check_suite_focus=true

Comment (taktv6) [reply .github/linters/.golangci.yml:3]: So the new CI system would not build certain parts of the project. 
Sorry, wont merge.

Comment (corny) [reply .github/linters/.golangci.yml:3]: It's just the linter that has been added. It did not exist before.

**Comment by taktv6** [comment]
Please make all actions work properly. Then I'm happy to merge.

**Comment by awlx** [comment]
Well the actions work, but the linter is really unhappy. It basically complains about every file, maybe this one shouldn't be used.

**Comment by awlx** [comment]
@taktv6 are we sure we want to test on MacOS or should that be excluded for now?

**Comment by awlx** [comment]
The circleci Error should go away as soon as it's merged ... as it's gone then.

**Review by corny — DISMISSED** [review]

**Comment by corny** [comment]
@taktv6 please merge

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+on: [push, pull_request]
+name: tests
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.16.x, 1.17.x]
```
Comment (taktv6) [inline .github/workflows/tests.yml:7]: Please 1.17 only here. 1.16 would not build anymore.

**Review by taktv6 — APPROVED** [review]

**Pull request merged**